### PR TITLE
tidy up tone code

### PR DESCRIPTION
**Pull request by danielt998**

### CODE REVIEW

File: src/main/java/hanziToAnki/DeckFactory.java [hunk 1]
```diff
@@ -53,65 +53,36 @@ private static String getOpeningHTMLTag(int tone) {
     }
 
     private static String getPinyinWithMarks(String syllable) {
```
Comment (IdiosApps) [inline src/main/java/hanziToAnki/DeckFactory.java:55]: Can we have some tests for this?

Comment (danielt998) [reply src/main/java/hanziToAnki/DeckFactory.java:55]: It's difficult and generally not recommended to test private methods. It should get tested implicitly from the test we have anyway, no?

Comment (IdiosApps) [reply src/main/java/hanziToAnki/DeckFactory.java:55]: Okay that makes sense, could we have a comment giving an example of what it does?

Comment (danielt998) [reply src/main/java/hanziToAnki/DeckFactory.java:55]: Do you think in some way it's not obvious from the method name? Could rename to something like convertSyllableFromNumbersToToneMarks?

Comment (IdiosApps) [reply src/main/java/hanziToAnki/DeckFactory.java:52]: That is a pretty decent suggestion. Before I was just getting lost in the code, but now I see:
`int tone = Integer.parseInt("" + syllable.charAt(syllable.length() - 1));` 
and seeing that with your new method name makes me realise what's going on.

How about this:
convertNumberedSyllableToAccentedSyllable

Also, I think we're fine to test this when it's moved to "ToneHelper.java" - https://github.com/danielt998/HanziToAnki/blob/master/src/main/java/hanziToAnki/ToneHelper.java

**Comment by IdiosApps** [comment]
Lovely

**Pull request merged**